src/cryptoadvance/specter/user.py

```diff
@@ -100,6 +100,12 @@ def set_explorer(self, specter, explorer):
         self.config["explorers"][specter.chain] = explorer
         self.save_info(specter)
 
+    def set_fee_estimator(self, specter, fee_estimator, custom_url):
```

Comment:
specter is passed in at every call. Can't we simplify that with passing it in at initialisation-time?